CITATION.cff

```diff
@@ -0,0 +1,39 @@
+# following https://github.com/citation-file-format/citation-file-format/blob/main/schema-guide.md
+
+abstract: "To facilitate a more low-barrier fitness-for-use evaluation of new datasets, we developed a data dashboard that allows the user to visualize metadata distribution information and sample images efficiently and with no coding required. Our current version is focused on plant and animal images, though for gathering metadata distribution statistics images are optional."
```

Comment:
```suggestion
abstract: "To simplify the assessment of new datasets for usability, we develop a data dashboard that allows the user to visualize metadata distribution information and sample images efficiently and with no coding required. Our current version is focused on plant and animal images, though images are optional for gathering metadata distribution statistics."
```
Is this wording a bit simpler?

Reply:
What about ...
```suggestion
abstract: "To simplify the fitness-for-use evaluation of new datasets, we developed a data dashboard that allows the user to visualize metadata distribution information and sample images efficiently and with no coding required. Our current version is focused on plant and animal images, though images are optional for gathering metadata distribution statistics."
```

Reply:
_Usability_ is not the same as _fitness-for-use_, so I don't think this is a good substitution. If you really wanted to make the sentence or statement "simpler", you could say _To make fitness-for-use evaluation for datasets easier_. But _easier_ could mean anything, so I'd argue carries little information to a reader. You could elaborate a little more, such as _To untether effective fitness-for-use evaluation for datasets from complex software deployment or learning burdens, ..._.

Finally, in the last sentence, I moved the _images are optional_ to **after** _gathering metadata distribution statistics_ for a reason. If it precedes it, it can easily be misread as images actually being the source of gathering metadata statistics. You could perhaps argue that simply changing the order doesn't fully address this potential for confusion, then suggest something else other than restoring the original order.

Reply:
I see what you mean on _usability_. _Fitness-for-use_ just a phrase that feels clunky to me. Perhaps _suitability_ is a more (fit-for-use/suitable) substitution?

On the last sentence, it reads more naturally to have the subject at the start of the second clause. We could rewrite to avoid the confusion about the images vs. metadata stats sourcing.

I think replacing first-person phrases helps the reading and focus on the dashboard itself as well.

```suggestion
abstract: "To simplify new dataset evaluation for suitability, this data dashboard allows the user to visualize metadata distribution information and sample images efficiently without coding. The current focus is on visualizations for datasets containing metadata of plant and animal images, though images are not used to gather metadata distribution statistics and are optional for displaying samples."
```

Reply:
LGTM!